File: eureka/ios/nativeLib/nativeLib/Database/dbTables/SessionsTable.swift
//
//  SessionsTable.swift
//  LifePlus
//
//  Created by work on 10/09/20.
//

import Foundation

public class SessionsTable {
    public static let TABLE_NAME = "sessions"

    public class Cols {
      public static let ID: String = "id"
      public static let USER_ID: String = "user_id"
      public static let DEVICE_ID: String = "device_id"
      public static let LOGIN_DATE: String = "login_date"
      public static let LOGOUT_DATE: String = "logout_date"
      public static let AUTH_TOKEN: String = "auth_token"
      public static let REFRESH_TOKEN: String = "refresh_token"
      public static let GATEWAY_TOKEN: String = "gateway_token"
    }
}

File: eureka/android/library/src/main/java/com/lifeplus/Database/dbTables/SessionsTable.java
package com.lifeplus.Database.dbTables;

public class SessionsTable {
    public static final String TABLE_NAME = "sessions";

    public static class Cols {
        public static final String ID = "id";
        public static final String USER_ID = "user_id";
        public static final String DEVICE_ID = "device_id";
        public static final String LOGIN_DATE = "login_date";
        public static final String LOGOUT_DATE = "logout_date";
        public static final String AUTH_TOKEN = "auth_token";
        public static final String REFRESH_TOKEN = "refresh_token";
        public static final String GATEWAY_TOKEN = "gateway_token";
    }
}
